tests/Behat/README.md

```
@@ -64,34 +72,24 @@ A new Redmine version could introduce new REST-API endpoints.
 Tests for this endpoint should not be run on older Redmine versions.
 This can be handled on the `scenario` or `feature` layer.
 
-1. Tag features or scenarios e.g. with `@since60000`.
+1. Tag features e.g. with `@since70000`.
 
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)